plugin/collectd-setup

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# collectd-setup Startup script to setup collectd-mlab switch configuration.
+# chkconfig: - 85 16
+# description: collectd-setup performs auto-discovery of the local switch
+#   configuration to complete collectd setup. collectd-setup should run before
+#   collectd.
+
+
+# NOTE: Perform SNMP auto-discovery for switch configuration.
+# If we cannot update the snmp community string, then the script should halt.
+COLLECTD_SNMP="/etc/collectd-snmp.conf"
+COMMUNITY="/home/mlab_utility/config/snmp.community"
+SWITCH="s1.${HOSTNAME#*.}"
+
+function start() {
+
+    # If the community string file is missing or zero size.
+    if ! test -s "${COMMUNITY}" ; then
```

Comment:
No, it was just my not recognizing/forgetting that the file $COMMUNITY was originally put in place by Ansible.  I had the mistaken impression when reading this script that $COMMUNITY was created by the script.  Now I understand. Thanks.